databases/week2/School.sql

```diff
@@ -0,0 +1,3 @@
+-- CREATED SCHEMA & TABLES USING MODEL WIZARD
+CREATE INDEX nameIndex ON student (name);
+ALTER TABLE class ADD status ENUM('not-started', 'ongoing', 'finished') NOT NULL;
```

Comment:
A default value is needed when you add a new field to the table

Reply:
Column altered to set the default value

Reply:
In the Library ER diagram
- deleted borrowing_id as member_id tracks it already
ALTER TABLE members
DROP COLUMN borrowing_id;
- rack_id is FK referring to id in book_location table

Reply:
I see, please rename the column rack_id to book_location_id
It will be what will be looking for if I was supposed to write a query.

Reply:
Changed the column name
ALTER TABLE books RENAME COLUMN rack_id TO book_location_id;